### PR TITLE
Updated guide

### PR DESCRIPTION
**Pull request by philip-nielsen**
Updated the README.md file, the check_prerequisites.sh and the run.sh script, to make it more clear to the user.

The kubectl provision cas command may fail due to local firewall settings. This can be fixed by opening port 8081/TCP on your local firewall. This may need to be written somewhere, maybe on the provision page. 
It is also explained in the tutorial that you need a Gitlab token, with read registry access. However, you also need one with write access, and this is not shown if you follow the link to "how to create a token".

### CODE REVIEW

**Review by scontainyourself — COMMENTED** [review]
Thanks for the changes! :+1: I left you some comments, which would be good if you incorporated. One more thing I am thinking... Maybe also add a general tip (e.g., using `> ` in md?) regarding using `./run.sh --help` to see more options and how to change the versions of the resources and images being used.  (`CAS_VERSION` also need to be added to the usage message of `run.sh --help`)

File: check_prerequisites.sh [hunk 1]
```diff
@@ -155,6 +156,7 @@ echo -e "${BLUE}Checking if the CAS '$CAS' in namespace '$CAS_NAMESPACE' is inst
 if ! (kubectl get cas "$CAS" -n "$CAS_NAMESPACE" )
 then
     echo -e "${RED}It seems that CAS '$CAS' in namespace '$CAS_NAMESPACE' is not yet running!${NC}"
+    echo -e "- ${ORANGE}- You can install a cas as follows: kubectl provision cas $CAS $CAS_NAMESPACE -v${NC}"
```
Comment (scontainyourself) [inline check_prerequisites.sh:159]: -n is missing.

File: README.md [hunk 2]
```diff
@@ -110,26 +110,32 @@ to which you want to deploy your Hello World application.
...
   kubectl get sgxplugin
   ```
 
-  > **NOTE:** If you are not authorized to use `default` Kubernetes namespace, you will have to ask somebody with access to install the [SCONE SGX Plugin](https://sconedocs.github.io/helm_sgxdevplugin/) 
+  > **NOTE:** If you are not authorized to install the services, you will have to ask somebody with access to install the [SCONE SGX Plugin](https://sconedocs.github.io/helm_sgxdevplugin/) 
   service and the [SCONE LAS](https://sconedocs.github.io/helm_las/)
   service. 
   > Although all the `kubectl` (as well as the `helm` and `run.sh`) commands in the rest of this tutorial also are using the `default` namespace, you can in those cases use the corresponding command line options for specifying another namespace, where you do have access rights.
 
+  - You will also need access to a CAS running in your kubernetes cluster for the course of this tutorial. If you do not yet have access to one, you can create a CAS using our kubectl plugin kubectl provision:
```
Comment (scontainyourself) [inline README.md:133]: Shoudl there really be a - here?

**Review by scontainyourself — COMMENTED** [review]
Just some minor comments. Then we are good to go.

File: README.md (hunk 2 above)
Comment (philip-nielsen) [reply README.md:133]: There is one above, in the section about SGX and LAS

**Review by scontain — APPROVED** [review]

**Pull request merged**